src/routes/calendar/+page.svelte

```
@@ -0,0 +1,203 @@
+
+<svelte:head>
+    <title>カレンダー — Anipolis</title>
+</svelte:head>
+
+<div class="page-container">
+    <!-- カレンダー本体 -->
+    <div class="feed-column">
+        <div class="card calendar-card">
+            <!-- ヘッダー：月ナビゲーション -->
+            <div class="calendar-header">
+                <button class="btn btn-ghost calendar-nav-btn" onclick={prevMonth} aria-label="前の月">
```

Biome (2.4.13)

[error] Provide an explicit type attribute for the button element.

(lint/a11y/useButtonType)